core/metadata.js

```diff
@@ -407,7 +408,7 @@ function upToDate(doc /*: Metadata */) /*: Metadata */ {
 
   return _.assign(clone, {
     errors: undefined,
-    sides: { local: rev, remote: rev }
+    sides: { rev, local: rev, remote: rev }
```

Comment:
I'd have expected `target: rev`. Why is it just `rev` here?

Reply:
Good catch. It's a leftover from a previous version of the PR in which `target` was still `rev`.
And Flow didn't catch this…

Reply:
Actually, the function is not used anymore. I've removed it.